responders/src/API/EMBC.Responders.API/Controllers/RegistrationsController.cs

```diff
@@ -347,15 +347,24 @@ public class RegistrationsMapping : Profile
         public RegistrationsMapping()
         {
             CreateMap<SecurityQuestion, ESS.Shared.Contracts.Submissions.SecurityQuestion>()
+                .ForMember(d => d.AnswerIsMasked, opts => opts.MapFrom(s => s.Answer.Contains("*")))
+                .ReverseMap()
+                ;
+
+            CreateMap<Address, ESS.Shared.Contracts.Submissions.Address>()
+                .ForMember(d => d.Community, opts => opts.MapFrom(s => s.CommunityCode))
+                .ForMember(d => d.Country, opts => opts.MapFrom(s => s.CountryCode))
+                .ForMember(d => d.StateProvince, opts => opts.MapFrom(s => s.StateProvinceCode))
                 .ReverseMap()
+                .ForMember(d => d.City, opts => opts.Ignore())
                 ;
 
             CreateMap<EvacueeProfile, ESS.Shared.Contracts.Submissions.RegistrantProfile>()
                 .ForMember(d => d.SecurityQuestions, opts => opts.MapFrom(s => s.SecurityQuestions))
                 .ForMember(d => d.RestrictedAccess, opts => opts.Ignore())
                 .ForMember(d => d.IsMailingAddressSameAsPrimaryAddress, opts => opts.Ignore())
-                .ForMember(d => d.MailingAddress, opts => opts.Ignore())
-                .ForMember(d => d.PrimaryAddress, opts => opts.Ignore())
+                .ForMember(d => d.MailingAddress, opts => opts.MapFrom(s => s.MailingAddress))
```

Comment:
These can be deleted, automapper will match the names automatically, you just need a map between the address DTOs